### PR TITLE
Add tabs component

### PR DESCRIPTION
**Pull request by irustm**
review in #6

### CODE REVIEW

File: snippets/tabs-component.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        Third tab body
+    </app-tab>
+ </app-tabs>
+
+
+# Links
+https://stackblitz.com/edit/angular-mm5gwg
+
+# ComponentCode
+  import { Component, Input, AfterContentInit, ContentChildren, QueryList } from '@angular/core';
```
Comment (thekiba) [inline snippets/tabs-component.md:26]:
```suggestion
import { Component, Input, AfterContentInit, ContentChildren, QueryList } from '@angular/core';
```

File: snippets/tabs-component.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        First tab body
+    </app-tab>
+    <app-tab title="Second tab">
+        Second tab body
+    </app-tab>
+    <app-tab title="Third tab">
+        Third tab body
+    </app-tab>
+  </app-tabs>
+  `,
```
Comment (thekiba) [inline snippets/tabs-component.md:42]: need add extra indent for template

File: snippets/tabs-component.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    if(activeTab) {
+      this.selectTab(activeTab);
+    } else {
+       this.selectTab(this.tabs.first);
+    }
+  }
+  
+  selectTab(tab){
+    this.tabs.forEach(tab => tab.active = false);
+     tab.active = true;
```
Comment (thekiba) [inline snippets/tabs-component.md:103]:
```suggestion
    tab.active = true;
```

File: snippets/tabs-component.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    <app-tab title="First tab">
+        First tab body
+    </app-tab>
+    <app-tab title="Second tab">
+        Second tab body
+    </app-tab>
+    <app-tab title="Third tab">
+        Third tab body
+    </app-tab>
+ </app-tabs>
```
Comment (thekiba) [inline snippets/tabs-component.md:19]: Need add description and wrap code

**Review by thekiba — CHANGES_REQUESTED** [review]

**Comment by NothingEverHappens** [comment]
Closing after talking to @irustm, it's cool, but would be really hard to fit the format

**Pull request closed without merging**